caravel/assets/javascripts/components/CopyToClipboard.jsx

```diff
@@ -29,17 +31,42 @@ export default class CopyToClipboard extends React.Component {
     this.onMouseOut = this.onMouseOut.bind(this);
   }
 
+  componentWillMount() {
+    this.getShortUrl();
+  }
+
   onMouseOut() {
     // delay to avoid flash of text change on tooltip
     setTimeout(this.resetTooltipText, 200);
   }
 
+  getShortUrl() {
```

Comment:
Seems like `CopyToClipboard` should be more generic, isn't it used in other places that have nothing to do with shortening URLs?